src/commonmark-react-renderer.js

```diff
@@ -21,16 +21,18 @@ var defaultRenderers = {
 
     List: function List(props) {
         var tag = props.type === 'Bullet' ? 'ul' : 'ol';
-        return createElement(tag, (
-            props.start !== null &&
-            props.start !== 1 &&
-            { start: props.start.toString() } || null
-        ), props.children);
+        var attrs = { key: props.key };
+
+        if (props.start !== null && props.start !== 1) {
+            assign(attrs, { start: props.start.toString() });
```

Comment:
Hrm, this won't work, I think? If I have two lists in the same container, both starting at `2`?